websites/P/Platzi/presence.ts

```diff
@@ -0,0 +1,504 @@
+const presence = new Presence({
+  clientId: '834628404233240628'
+}),
+estimatedTime = Math.floor(Date.now() / 1000);
+
+let available_logos: Array<string>, 
+schools: Array<string>,
+titleToID: Record<string, string>,
+categoriesEventListener = false,
+activeCategory = '';
+
+fetch('https://cdn.jsdelivr.net/gh/brunoocal/PlatziPreMID/coursesData.json')
```

Comment:
Okay for what purpose you need that? Because for now, as I see, I don't accept it at all.

Reply:
Yes, the fetch is to check what logos are available in the Discord app assets, because it only leaves room for 300 and there is more than that ...

Should I remove that functionality?

Reply:
yeah...